schema/base/bathymetry.yaml

```diff
@@ -0,0 +1,23 @@
+---
+"$schema": https://json-schema.org/draft/2020-12/schema
+title: bathymetry
+description: topographic representation of the ocean floor 
+type: object
+properties:
+  id:
+    "$ref": ../defs.yaml#/$defs/propertyDefinitions/id
+  geometry:
+    unevaluatedProperties: false
+    oneOf:
+      - "$ref": https://geojson.org/schema/Polygon.json
+      - "$ref": https://geojson.org/schema/MultiPolygon.json
+  properties:
+    unevaluatedProperties: false
+    allOf:
+      - "$ref": ../defs.yaml#/$defs/propertyContainers/overtureFeaturePropertiesContainer
+      - "$ref": ../defs.yaml#/$defs/propertyContainers/levelContainer
+      - "$ref": ../defs.yaml#/$defs/propertyContainers/cartographyContainer
+    required:
+      - elevation
+    properties:
```

Comment:
1. Should we add a relationship property to reference the feature to the water body it is providing the bathymetry for?
2. Should we add a property to specify what type of water body it pertains to?

I'm not too sure about the second one, but adding it for discussion. Might be useful to know if something is ocean or inland at least?

For the first one, the ID reference, it would only pertain to "things that exist in the base theme" - we don't have features for ocean water - so it would be optional.

Just throwing out ideas, but we might imagine something like:

```yaml
---
id: foo
properties:
  theme: base
  type: bathymetry
  depth: 100
  water_location: inland  # Can be "ocean" or "inland"
  water_id: bar           # Omit if there's no matching water feature, otherwise this is ID of a base/water feature
```

Reply:
`water_location` might be useful, but not sure how we can relate them to water bodies since the cross cut so many?